src/stories/pages/ExperimentsPage/ExperimentsPage.js

```diff
@@ -0,0 +1,174 @@
+    userService.getCurrentUser(setLogInState).catch((e) => {
+      setLogInState({ isLoggedIn: false });
+      setError(e);
+    });
+  }, [logInState.isLoggedIn]);
+
+  useEffect(() => {
+    getAllExperiments().then((response) => {
+      if (response.status === 200) {
+        // TODO #3.1(fernandowinfield): Filter ongoing and inactive experiments
+        // and set state for both instead of setting state for all.
+        setAllExperiments(response.data);
+      } else {
+        setError(response.error);
+      }
+    });
+  }, []);
+
+  // TODO #3.2(fernandowinfield): Map ongoingExperiments instead of all.
+  const ongoingExperimentsSlides = allExperiments ?
+    allExperiments.map(
+      (
+        // TODO #2.1(fernandowinfield): add/remove fields below as needed.
+        { title = 'Title of Experiment',
+          shortDesc = 'Short description of experiment',
+          opened = 'mm-dd-yyyy',
+          admin = 'Admin name',
+          researchers = ['Researcher Name'],
+        },
+        i,
+      ) =>
+        <Card
+          modifierClasses='card--active'
+          title={title}
+          body= {shortDesc}
+          opened={opened}
+          admin={admin}
+          researchers={researchers.join(', ')}
+          key={i}
+        />,
+    ) :
+    null;
+
+  // TODO #3.3(fernandowinfield): Map inactiveExperiments instead of all and
+  // add ternary operator like in the ongoingExperiments mapping (to handle
+  // the case when there are no inactive experiments).
+  const inactiveExperimentsSlides = PLACEHOLDER_EXPERIMENTS.map(
+    (
+      // TODO #2.2(fernandowinfield): here too, add/remove fields below as
+      // needed.
+      { title = 'Title of Experiment',
+        shortDesc = 'Short description of experiment',
+        opened = 'mm-dd-yyyy',
+        admin = 'Admin name',
+        researchers = ['Researcher Name'],
+      },
+      i,
+    ) =>
+      <Card
+        modifierClasses='card--inactive'
+        title={title}
+        body= {shortDesc}
+        opened={opened}
+        admin={admin}
+        researchers={researchers.join(', ')}
+        key={i}
+      />);
+
+  return error ? (
+    <div>{error.errorCode + ': ' + error.errorMessage}</div>
+  ) : logInState.isLoggedIn ? (
+    <UserContext.Provider value={logInState.user}>
+      <div>
+        <HorizontalTitle
+          eyebrow={'Overview'}
+          title={'Experiments'}
+          content={
+            'Here\'s an overview of what\'s going on your application. ' +
+            'You can review your active experiments, ' +
+            'check reports and analyze real-time data.'
+          }
+        />
+
+        {/* Ongoing experiments */}
+        <div className='experiments__ongoing-section'>
+          <div>
+            <h6 className='experiments__section-header'>
+              ONGOING EXPERIMENTS
+            </h6>
+            <AddExperiment buttonText={'+'}
+              buttonModifierClasses={'button--circular-small'} />
```

Comment:
Small thing I noticed, the button gets distorted on hover. You might want to override/adjust the styles accordingly.